app/publishing/index.js

```diff
@@ -3,7 +3,8 @@ const publish = require('./publish')
 const saveRequest = require('./save-request')
 
 const publishStatement = async (request) => {
-  const response = await publish(request.email, request.filename)
+  const personalisation = { schemeName: request.scheme.name, schemeShortName: request.scheme.shortName, schemeYear: request.scheme.year, schemeFrequency: request.scheme.frequency.toLowerCase(), businessName: request.businessName }
```

Comment:
I would suggest moving this into a sep. file/func for coupling reasons

Regardless of whether you decide to split this out, I think this needs unit tested as I had no idea this mapping took place in the integration tests

Reply:
Agreed.  Split and tests added.